packages/core/src/storages/dataset.ts

```diff
@@ -282,6 +284,19 @@ export class Dataset<Data extends Dictionary = Dictionary> {
         }
     }
 
+    /**
+     * Save the entirety of the dataset's contents into one file within a key-value store.
+     *
+     * @param key The name of the value to save the data in.
+     * @param options An optional options object where you can provide a `keyValueStoreName` and a `contentType` for the output.
+     */
+    async exportToValue(key?: string, options: RecordOptions & { keyValueStoreName?: string } = {}) {
```

Comment:
i think this could be mentioned in the examples section, otherwise it will be a bit hidden (documented only in API docs)